drams/calva_getting_started/hello_paredit.clj

```diff
@@ -62,6 +62,9 @@
 ;; Move form-by-form using *Paredit Sexp Forward*
 ;; and *Paredit Sexp Backward*
 
+;; Note: For Mac it is alt+right/left and
+;; for Windows and Linux it is ctrl+ right/left
+
```

Comment:
I think it was a good idea you had, to mention the issue.

```suggestion
;; Note: Despite what the command palette is showing, the
;; Keyboard shortcuts for For Mac are alt+right/left and
;; for Windows and Linux they are ctrl+right/left
;; See https://github.com/BetterThanTomorrow/calva/issues/1161
```

Reply:
kewl